### PR TITLE
Remove tables_count deprecated field

### LINKED ISSUE #4127: Remove tables_count field deprecated
As it seems like the field tables_count of user model has been deprecated and it's not updated anymore it should be removed with a migration and clean the code in order to remove any reference

/cc @Kartones

### PR DESCRIPTION
**Pull request by Kartones**
Fixes #4127

### CODE REVIEW

**Comment by Kartones** [comment]
@ethervoid CR please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5353/))

File: db/migrate/20150724121847_remove_table_count_from_users.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Sequel.migration do
```
Comment (ethervoid) [inline db/migrate/20150724121847_remove_table_count_from_users.rb]: As we did with `dynamic_field_enabled` field, you should put the ward to  avoid the cache of active record:

http://blog.codeship.com/rails-migrations-zero-downtime/

Comment (Kartones) [inline db/migrate/20150724121847_remove_table_count_from_users.rb]: but is not used/queried anywhere, will it break?

Comment (ethervoid) [inline db/migrate/20150724121847_remove_table_count_from_users.rb]: If no one use it should not break.

**Comment by ethervoid** [comment]
:+1:

**Pull request merged**